examples/langevin/compare.py

```diff
@@ -0,0 +1,20 @@
+#!/bin/env python
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.stats import ks_2samp
+
+r_mc, g_mc = np.loadtxt('rdf_mc.dat', unpack=True, usecols=(0,1))
+r_ld, g_ld = np.loadtxt('rdf_ld.dat', unpack=True, usecols=(0,1))
+
+stats = ks_2samp(r_mc, r_ld)
+if stats[1] > 0.99:
+   print('p-value = {} (no significant difference)'.format(stats[1]))
+else:
+   print('p-value = {} (significant difference)'.format(stats[1]))
+
+plt.plot(r_mc, g_mc, label='Monte Carlo', linewidth=10, alpha=0.5)
```

Comment:
Very nice check to perform during the test suite, i.e. `ctest`. The plotting step should be skipped.

Reply:
Before we go on with this step, I want to conduct some noise to one of the distributions to check how much difference there would have to be before the [Kolmogorov-Smirnov test](https://en.wikipedia.org/wiki/Kolmogorov%E2%80%93Smirnov_test) consider the the two distributions significantly different given a significance level of 99%.